fastods/src/main/java/com/github/jferard/fastods/style/PageStyleBuilder.java

```diff
@@ -123,7 +123,7 @@ public PageStyleBuilder backgroundColor(final Color color) {
     public PageStyle build() {
         if (this.masterPageStyle == null) {
             this.masterPageStyle =
-                    new MasterPageStyle(this.name, this.name, this.header, this.footer);
+                    new MasterPageStyle(PageStyle.DEFAULT_MASTER_PAGE_NAME, this.name, this.header, this.footer);
```

Comment:
I'm not sure this is useful. The `DEFAULT_MASTER_PAGE_NAME` is used by the `DEFAULT_MASTER_PAGE_STYLE` used by the `DEFAULT_TABLE_STYLE`. Hence the modification of the name should be propagated to the XML file.

Reply:
This seems to be crucial for Excel, the first has to be "Default" (same goes for the DefaultMasterPageName), so in spirit of DRY i just used `PageStyle.DEFAULT_MASTER_PAGE_NAME` but there could also be a separate constant if you prefer

Reply:
Okay. I think there's a bug here in the current version. In the `a_hello_world_example.ods` (IT test), the `content.xml` file reads:

    <style:style style:name="ta1" style:family="table" style:master-page-name="DefaultMasterPage">

But there is no `DefaultMasterPage` style in `styles.xml`, just a `Mpm1`. You solved this issue by naming all master page styles `Default` but we need many master page styles in complex documents (this is why the name can't be a constant). I will investigate to see why the `PageStyle.DEFAULT_MASTER_PAGE_NAME` is not added to the document.

Reply:
Ok, I'm sure about this implications, but as far as I gathered before the change, the first two where always the same (name and master-page-name) and there was no way to apply different values...

Reply:
Here's my suggestion: I'll accept a pull request with the work on forms we discussed above and the name change (`DefaultMasterPage` to `Master`). Then I'll open an issue on this question (`Default` master page style is absent of `styles.xml`). Once this is fixed, we can move forward on the Excel compatibility.

Reply:
Sounds good!

Reply:
What bothers me is that if, I write the forms part, your PR will shrink to a name change. I'll accept it anyway, because your insights have been helpful, but it may be frustrating for you. If you agree, just revert the changes on the other files and push again. Otherwise I'll close this PR (and credit you in the change log in both cases).

Reply:
No worries i'm just intersted in Excel compatibility :)

Reply:
Sorry, i'm not really well with github, how do i revert the changes?

Reply:
> No worries i'm just intersted in Excel compatibility :)
If that's all, just close the PR and I'll do the job, I'll credit you in the change log for the suggestions.

Reply:
Alright so the PageStyle.DEFAULT_MASTER_PAGE_NAME must be "Default" this must also be the first parameter in the PageStyleBuilder MasterPageStyle and the forms must only be included if set, I think that's it, thank you for your help!